Unbound.xs

```diff
@@ -597,7 +604,7 @@ void
 DESTROY (DNS__Unbound__Context* dub_ctx)
     CODE:
         _DEBUG("%s", __func__);
-
+        if (getpid() != dub_ctx->pid) return;
```

Comment:
I think this isn’t quite the right approach since we still want to decrement the refcount of `ctx->queries_hv`.

I’m also wondering what’ll come of https://github.com/NLnetLabs/unbound/issues/775.